src/context.rs

```diff
@@ -371,6 +381,15 @@ impl ContextBuilder {
     }
 }
 
+/// Attempts to terminate the [`ggez::event::run()`](fn.run.html) loop by requesting a
+/// [`quit_event`](EventHandler::quit_event) at the very start of the next frame. If this event
+/// returns `Ok(false)`, then [`Context.continuing`](struct.Context.html#structfield.continuing)
+/// is set to `false` and the loop breaks.
+#[deprecated(since = "0.8.0", note = "Use `ctx.request_quit` instead.")]
+pub fn quit(ctx: &mut Context) {
+    ctx.request_quit();
+}
+
```

Comment:
Here we could even keep the old implementation of just setting `ctx.continuing` to `false`. This would lead to no `quit_event` being triggered, as it is in ggez 0.7 and in all versions prior.

On the other hand I like that this is no longer possible... but since this is a deprecated function it might make sense to keep the deprecated behavior still.

Reply:
When was `quit_event` even called before

Reply:
Only when winit requested a quit, when the user tried to close the window for example.